src/modules/LanUdevAuto.ycp

```diff
@@ -182,12 +182,11 @@ global string FindBy(boolean(map, string) validator, string attr)
 {
     string ret = "";
 
-    foreach(integer index, map devmap, (map<integer, map<string, any> >)LanItems::Items,
+    foreach(map hwmap, ReadHardware("netcard"),
     {
-        // we're interested in configured devices only => return ifcfg name
```

Comment:
bcs the patch didn't worked in both cases (installation / running system) properly.